_posts/2016/10/2016-10-20-community-call.md

```diff
@@ -1,18 +1,23 @@
 ---
 layout: post
-authors: ["Kari L. Jordan"]
+subheadline:
 title: "Community Call on Assessment"
-date: 2016-10-20-
-time: "17:30"
-category: ["Assessment", "Community"]
+teaser: "Discussion of our workshop survey results"
+header:
+ image_fullwidth: "light-blue-wood-texture.jpg"
```

Comment:
Is this copy/pasted from the DC blog?  We don't seem to have that image in this repo.